products/logging/terraform.yaml

```diff
@@ -38,6 +38,8 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     properties:
       metricDescriptor.labels.valueType: !ruby/object:Overrides::Terraform::PropertyOverride
         custom_flatten: 'templates/terraform/custom_flatten/default_if_empty.erb'
+      metricDescriptor.unit: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
```

Comment:
What does the API default to? As far as I know we're trying to avoid Optional+Computed when possible - might be better to just set a default.

Reply:
Oh, sure, good to know.  I can default it to "1".  Thanks!